lib/virtfs/xfs/file_class.rb

```diff
@@ -0,0 +1,224 @@
+module VirtFS::XFS
+  class FS
```

Comment:
Agreed.  Also based on previous PRs for the other retypes.  @movitto I will make this comment on your PRs as well.